abstract.tex

```diff
@@ -31,17 +31,35 @@
 
-Moltres is an open source coupled neutronics/thermal hydraulics simulation tool for simulating MSRs \cite{lindsay_introduction_2018}. It is under active development at the University of Illinois. It is an application code built using the MOOSE finite element framework \cite{gaston_moose:_2009}. Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations. As a multi-group neutronics solver on a coarse mesh, Moltres imposes relatively low computational loads compared to continuous energy neutronics solvers and computational fluid dynamics thermal-hydraulic solvers employed by other researchers in the study of MSRs. This allows for steady-state, transient and accident analyses with relatively low computational requirements.
+The development of novel simulation tools for MSRs is an essential step towards understanding the behaviour of MSRs under various conditions, one of which is the changing fuel compositions across the lifespan of MSRs.
+The fuel composition in an MSR continuously evolves over time as online refueling replenishes fissile material while removing fission product poisons. The impact of changing fuel composition on the safety parameters such as the reactivity feedback coefficient must be studied extensively.
 
-This abstract presents the outline for the code-to-code verification of Moltres. The reactor model used for this verification is the Molten Salt Fast Reactor. The MSFR is a reference design for a fast-spectrum molten salt reactor developed under the EVOL and SAMOFAR projects\cite{serp_molten_2014}. There is abundant MSFR parameter and simulation data available from various authors \cite{fiorina_modelling_2014} \cite{pettersen_coupled_2016} for the reference model to perform the code-to-code verification.
+Moltres is an application code built in the MOOSE finite element framework \cite{gaston_moose:_2009}.
+Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations.
+MOOSE is a parallel framework and it has an adaptive meshing scheme.
+Therefore, reactor simulations with large complex meshes can scale effectively when run on multiple cores. 
 
-Moltres requires neutron group constant data from a neutron transport code. For this study, Serpent is used to generate the group constant data. Serpent \cite{leppanen_serpent_2015} is a continuous-energy Monte Carlo code used in numerous reactor physics applications. The inputs consist of MSFR reference model parameters, neutron energy group bounds, a nuclear data library, and the temperatures at which the group constants are to be generated. Six neutron group constant data is generated for the six-group neutron diffusion equation.
+The Molten Salt Fast Reactor (MSFR) is the MSR model used for the code-to-code verification.
```

Comment:
Why so much passive voice??!?!?!?!!?
In #2 I expressed that there was too much passive voice in the first draft.
This PR (which was supposed to address #2, not #3 by the way... ) seems to go entirely in the other direction, adding passive voice where active voice used to appear.... What is goign on? I am going to stop reading here. Please fix this before the next request for a review.